vendor/engines/secure_rooms/app/views/secure_rooms/card_readers/new.html.haml

```diff
@@ -0,0 +1,15 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: "secure_rooms"
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "card_readers"
+
+%h2= @product
+%h3= text("views.admin.secure_rooms.card_readers.new.title")
+
+= simple_form_for @card_reader, url: facility_secure_room_card_readers_path(current_facility, @product) do |f|
```

Comment:
updated, but re: sharing code--one the one hand, these two files are now comically similar, but on the other hand, this is only duplication not triplication of a small amount of code and we have no budget, but then I guess even thinking about it is burning money soo ugh I'll just change it